spec/features/users/new_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+describe "the User Registration Page" do
+  before(:each) do
+
+  end
+
+  it "shows the fields to enter new user information" do
+    visit register_path
+    expect(page).to have_content("Register an Account")
+    expect(page).to have_field("user[name]", type: "text")
+    expect(page).to have_field("user[email]", type: "email")
+    expect(page).to have_button("Register")
+
+    fill_in "user[name]", with: "Thomas Smith"
+    fill_in "user[email]", with: "tsmith11@turing.edu"
+    
+    click_on("Register")
+    
+    x = User.find_by(name: "Thomas Smith")
```

Comment:
Nice work on this! I have received feedback not to use single character variables like `x`, but this is a really simple instance so I think we're ok.